doozerlib/operator_metadata.py

```diff
@@ -575,6 +575,65 @@ def operator(self):
         return self.runtime.image_map[self.operator_name]
 
 
+class OperatorMetadataLatestNvrReporter:
+    """Query latest operator metadata based on nvr and stream"""
+
+    @log
+    def __init__(self, operator_nvr, stream, runtime):
+        self.operator_nvr = operator_nvr
+        self.stream = stream
+        self.operator_name, self.operator_version = self.unpack_operator_nvr()
+        self.metadata_name = '{}-metadata-container'.format(self.operator_name)
+        self.runtime = runtime
+
+    @log
+    def get_latest_build(self):
+        regex = r'^{}-metadata-container-{}\.{}-(?P<release>[0-9]+)$'.format(
```

Comment:
Given above I would think this needs to change too

Reply:
solved here: https://github.com/openshift/doozer/pull/123/commits/066a47889b068316af499db4b49e5c9b869acb7a#diff-9206bb1bd248a870ebf5892a4dab3341R595-R602